content/documents/order-returns.mdx

```diff
@@ -3,11 +3,12 @@ type:           article
 title:          Introducing Order Returns
 description:    New set of features to enable all aspects of the Order Returns process.
 author:         mposthumus
-publishDate:    2022-05-16    
+publishDate:    2022-05-16
+updatedDate:    2022-05-16   
```

Comment:
don't know if it's a bug, but if an article is missing `updateddate` this is what displays in the UI 
![image](https://user-images.githubusercontent.com/10764202/172190803-8e9eb7c4-d0b9-49e8-8b54-bb8aaf523c98.png)

Reply:
looks like a bug, I'll get it logged.  thanks!